Resources/UnrarKit-Info.plist

```diff
@@ -15,10 +15,10 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>2.8-beta</string>
```

Comment:
I'll update it out of beta after we merge this request. I've got a scripted process for doing so.